web/src/App.js

```diff
@@ -1,10 +1,15 @@
 import React, { Component } from 'react';
 import './App.css';
+import {Search} from './Search';
+import {GetUser} from './GetUser';
+
 
 class App extends Component {
   constructor(props) {
     super(props);
-    this.state = { value: '' };
+    this.state = {
+      userName: null
+    }
 
     this.handleChange = this.handleChange.bind(this);
     this.handleSubmit = this.handleSubmit.bind(this);
```

Comment:
This `bind` is no longer necessary, as you use `handleSubmit = (userName) => {` notation below.